packages/salesforcedx-vscode-apex/snippets/apex.json

```
@@ -410,5 +409,54 @@
             "}"
         ],
         "description": "Create a mock for an http callout for apex testing"
+    },
+    "Is Blank": {
+        "prefix": "isb",
+        "body": [
+            "String.isBlank(${0})"
```

According to the [Apex documentation on the String class](https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_string.htm), isBlank(inputString) is a method that takes one argument of type String.  I would put a placeholder text inside the method so that users don't get misled into thinking that isBlank(inputString) is ok to use with zero arguments.

```suggestion
            "String.isBlank(${1:inputString})"
```